packages/currency/src/iso4217/data-iso4217.ts

```diff
@@ -0,0 +1,1356 @@
+/*
+	Follows ISO 4217, https://www.iso.org/iso-4217-currency-codes.html
+	See https://www.currency-iso.org/dam/downloads/lists/list_one.xml
+	Data last updated 2018-08-29
+*/
+/* eslint-disable spellcheck/spell-checker */
+export default [
```

Comment:
any reason not to no longer use the iso currency package ? would avoid us to require to update this list manually, even if I guess this won't change any time soon :)

Reply:
As we are getting closer to the protocol this must not change or with a new version of the request logic.
So we cannot depend on an external library.
I don't have the same concern on the ERC20 currencies as the number of decimal is define directly on the smart contract of the token.

Reply:
fair enough, but that means reviewing this list every now and then. Maybe we should subscribe to changes to the original repo we were using